src/SideNavigation/SideNavigation.js

```diff
@@ -1,204 +1,263 @@
-                                                className={`fd-side-nav__icon sap-icon--${item.glyph} sap-icon--l`}
-                                                role='presentation' />
-                                        ) : null}
-                                        {item.name}
-                                    </a>
-                                ) : null}
-
-                                {item.hasChild ? (
-                                    <ul
-                                        aria-expanded={this.state.itemStates[item.id]}
-                                        aria-hidden={!this.state.itemStates[item.id]}
-                                        className='fd-side-nav__sublist'
-                                        id={item.id}>
-                                        {item.child.map(ch => {
-                                            return (
-                                                <React.Fragment key={ch.id}>
-                                                    {ch.link ? (
-                                                        <Link
-                                                            className={this.getSubLinkClasses(ch.id)}
-                                                            key={ch.id}
-                                                            onClick={e => this.handleSelectChild(e, ch.id)}
-                                                            to={{ pathname: ch.link }}>
-                                                            {ch.name}
-                                                        </Link>
-                                                    ) : null}
-
-                                                    {ch.url ? (
-                                                        <a
-                                                            className={this.getSubLinkClasses(ch.id)}
-                                                            href={ch.url}
-                                                            key={ch.id}
-                                                            onClick={e => this.handleSelectChild(e, ch.id)}>
-                                                            {ch.name}
-                                                        </a>
-                                                    ) : null}
-                                                </React.Fragment>
-                                            );
-                                        })}
-                                    </ul>
-                                ) : null}
-                            </li>
-                        );
-                    })}
-                </ul>
-            </BrowserRouter>
+            <div
+                className={sideNavHeaderlasses}>
+                <h1 {...titleProps} className='fd-side-nav__title'>
+                    {title}
+                </h1>
+                {sideNavList}
+            </div>
```

Comment:
Do we need this `<div>` wrapper?  It seems odd that it's included when a title exists, but not otherwise.  It appears both styles tied to `fd-side-nav__group` already exist on `fd-side-nav__list` and `fd-side-nav__title` does not require a specific parent class.  Maybe just `<React.Fragment>` here instead?

Reply:
In this case I think we should try to match the Fundamental HTML which does include this `<div>`.